README.md

```diff
@@ -0,0 +1,71 @@
+# Kiwix Apple Custom Apps
+
+Kiwix Apple custom apps are iOS/macOS apps running [Kiwix for
+Apple](https://github.com/kiwix/apple) against a
+pre-configured ZIM file.
+
+This project contains data and scripts needed to create specific
+ custom Kiwix Apple apps.
+
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+
+## Custom app folder
+
+[In the repository](https://github.com/kiwix/kiwix-apple-custom),
+each custom configuration is isolated in a so called custom app
+folder. If you need to create a new one for a new custom app, you can
+do that easily using Github UI. Go to
+https://github.com/kiwix/kiwix-apple-custom/new/main, put the name
+of your app (no space, lowercase only) and add a slash at the end.
```

Comment:
What will does this exactly? It seems you use a feature of Github I'm not familiar this. Where is the behaviour defined?

Reply:
It has been taken from the [android custom repo](https://github.com/kiwix/kiwix-android-custom/blob/main/README.md?plain=1#L63-L65). I wasn't sure if it's necessary.
I removed this for now.